src/components/__tests__/Rockets.test.js

```diff
@@ -0,0 +1,9 @@
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../../redux/store';
+import Rockets from '../Rockets';
+
+test('home component match snapshot', () => {
+  const tree = renderer.create(<Provider store={store}><Rockets /></Provider>).toJSON;
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly make sure to solve the errors in order for your test cases to pass because for now they are failing.

- [ ] Kindly make sure to write unit tests for the `Missions` and `Profile` components as well.


   - ### TESTING ERRORS
  
    ![image](https://user-images.githubusercontent.com/87197412/224342780-db0e79f3-973c-4ca3-b36a-540c0a0f4408.png)